Add plenty support + whale emote to identify large ovens

chasdabigone opened the pull request.

chasdabigone committed:
mark whale trades with emote

fetch owners of large ovens to put a whale emote on their trades

chasdabigone committed:
Update main.py

Fitblip commented on main.py:
```diff
@@ -29,6 +29,25 @@ def send_discord(payload):
+
+    whaleSize = 100000 #size threshold
+    whaleSize = whaleSize * 1e18 #convert using mantissa
+    global whales
```
There's a few ways that might be better to do this instead of a global like this.

You can either pass around `whales` (returning it in this function, then passing it down into `fetch_contract_activity`, or you could define `whales = []` above `watch_for_changes` at the bottom of this file, then it should be accessible globally (in a way that doesn't use `global` - I generally discourage using this because there's a more discoverable way to do it).

Fitblip commented:
Nice nice, thanks for the PR! LMK if my review comments make sense or if you have any questions 👍

chasdabigone committed:
removed global variable from function to main body

chasdabigone committed:
add plenty support

added function to fetch plenty data
added function to pull quipu+plenty and set ophash
added plenty trades to handle_new_transfers
changed watch_for_changes to use the new functionality and save hash for both exchanges

chasdabigone committed:
added kusd/usdtz pair from plenty

added kusd/usdtz pair on plenty
added support for whale channel
fixed issue where API didnt filter by 'applied' transactions

Fitblip commented:
Awesome! Looks good to me. I might do some formatting updates in a follow-on PR but if this works let's get it into prod! :D
